requirements-dev.txt

```diff
@@ -16,7 +16,7 @@ pandas
 pillow
 psutil
 pydantic
-pytest
+pytest<7.0
```

Comment:
What happened here? Were there breaking changes?

Reply:
Yeah, they just released 7.0 a couple days ago and our test suite breaks. There wasn't a simple/obvious fix to the error I was seeing. I'll file an issue to investigate.